README.md

```diff
@@ -35,15 +35,15 @@ Make sure Xcode 9 is installed first.
 ### Homebrew
 
 ```sh
-$ brew tap yonaskolb/mint https://github.com/yonaskolb/mint.git
+$ brew tap yonaskolb/Mint https://github.com/yonaskolb/Mint.git
 $ brew install mint
 ```
 
 ### Make
 
 ```sh
-$ git clone https://github.com/yonaskolb/mint.git
-$ cd mint
+$ git clone https://github.com/yonaskolb/Mint.git
```

Comment:
I could also change the repo name to be lowercase. What do you think? I think I prefer uppercase, but could be swayed

Reply:
There is no reason not to keep the repository and package names capitalized. That is the way most projects are and I think it looks nicer.